modules/distr-tracing-tempo-update-olm.adoc

```diff
@@ -0,0 +1,9 @@
+//Module included in the following assemblies:
+//
+//* distr_tracing_install/dist-tracing-tempo-updating.adoc
+
+:_content-type: PROCEDURE
+[id="distr-tracing-tempo-update-olm_{context}"]
+= Updating the {TempoShortName} by using the OLM
```

Comment:
This isn't really a procedure, how about something like this?
```suggestion
:_content-type: CONCEPT
[id="distr-tracing-tempo-update-olm_{context}"]
= Automatic updates of the {TempoShortName} by using the OLM
```

Or maybe without "by using the OLM"
```suggestion
:_content-type: CONCEPT
[id="distr-tracing-tempo-update-olm_{context}"]
= Automatic updates of the {TempoShortName}
```

Reply:
Thank you so much. Applying your second suggestion manually.